controllers/user.go

```diff
@@ -322,41 +325,59 @@ func (c *ApiController) UpdateUser() {
 	if id == "" {
 		id = c.GetSessionUsername()
 		if id == "" {
+			logger.Error(goCtx, "missed parameter to update user",
```

Comment:
Same concern about operation name applies here.
We also need to add name of that missing parameter to error message.

Reply:
done